modules/kafka/src/main/scala/com/dimafeng/testcontainers/KafkaContainer.scala

```diff
@@ -1,51 +1,29 @@
 package com.dimafeng.testcontainers
 
 import org.testcontainers.containers.{KafkaContainer => JavaKafkaContainer}
+import org.testcontainers.utility.DockerImageName
 
-class KafkaContainer(confluentPlatformVersion: Option[String] = None,
-                     externalZookeeper: Option[String] = None) extends SingleContainer[JavaKafkaContainer] {
```

Comment:
Has this argument been removed on purpose? It doesn't seem to be deprecated in the Java lib

Reply:
Are you about `externalZookeeper`? 

If yes. Then by default `externalZookeeperConnect = null` in Java lib, so then we do not need pass any value, cause by default we will use `embeddedZookeeper` (because method `withEmbeddedZookeeper` returns `externalZookeeperConnect = null`).
And if anyone wants to use externalZookeeper they should use `configure` method (as written above the val `kafkaContainer` )

Reply:
Sorry for vague question, and yes, I was asking about `externalZookeeperConnect`. 

Thank you, it makes sense!